sBTC SIP

```diff
@@ -0,0 +1,166 @@
+# sBTC SIP
```

Comment:
Said elsewhere already, but please read [SIP-000](https://github.com/stacksgov/sips/blob/main/sips/sip-000/sip-000-stacks-improvement-proposal-process.md) for how to properly format a SIP.